components/scream/src/control/surface_coupling_export.cpp

```diff
@@ -22,21 +22,25 @@ void SurfaceCoupling::do_export (const bool init_phase)
   const bool scream_ad_run =
       (m_field_mgr->has_field("qv") && m_field_mgr->has_field("T_mid") &&
        m_field_mgr->has_field("p_mid") && m_field_mgr->has_field("pseudo_density") &&
-       m_field_mgr->has_field("precip_liq_surf") && m_field_mgr->has_field("precip_ice_surf"));
+       m_field_mgr->has_field("precip_liq_surf") && m_field_mgr->has_field("precip_ice_surf") &&
+       m_field_mgr->has_field("p_int") && m_field_mgr->has_field("phis") ); //PMC added
   if (scream_ad_run) {
     const int last_entry = m_num_levs-1;
     const auto& qv              = m_field_mgr->get_field("qv").get_view<const Real**>();
     const auto& T_mid           = m_field_mgr->get_field("T_mid").get_view<const Real**>();
     const auto& p_mid           = m_field_mgr->get_field("p_mid").get_view<const Real**>();
+    const auto& p_int           = m_field_mgr->get_field("p_int").get_view<const Real**>();
     const auto& pseudo_density  = m_field_mgr->get_field("pseudo_density").get_view<const Real**>();
     const auto& precip_liq_surf = m_field_mgr->get_field("precip_liq_surf").get_view<const Real*>();
     const auto& precip_ice_surf = m_field_mgr->get_field("precip_ice_surf").get_view<const Real*>();
+    const auto& phis = m_field_mgr->get_field("phis").get_view<const Real*>();
```

Comment:
Not at all important, but since I left a comment elsewhere I figured I would leave this comment too.  For code readability we could align the `=` sign for this line with all other lines above it.  Like you did with the addition of `p_int`.

Reply:
@PeterCaldwell I'll align this as well with my test fix (@AaronDonahue comment above)